notes-webapp/src/main/webapp/vue-app/notes-editor/components/NotesEditorDashboard.vue

```diff
@@ -383,7 +383,7 @@ export default {
             name: this.note.name,
             wikiType: this.note.wikiType,
             wikiOwner: this.note.wikiOwner,
-            content: this.note.content,
+            content: this.getBody() ? this.getBody() : this.note.content,
```

Comment:
here you call twice this.getBody.
Is it possible to call it once ?
you could use this.getBody() || this.note.content 
I thought it work, but please check.
Else you can use a variable